res/client/login.css

```diff
@@ -0,0 +1,4 @@
+QWidget#header
+{
+    background-color: rgba(255,255,255,255);
```

Comment:
According to [Qt docs](http://doc.qt.io/qt-4.8/stylesheet-reference.html): "The rgb() or rgba() syntax can be used with integer values between 0 and 255, or with percentages. "